### PR TITLE
Add basic l2tp test

### PR DESCRIPTION
**Pull request by emanchado**
Onboard basic L2TP test.

### CODE REVIEW

File: networking/tunnel/l2tp/basic/Makefile [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	@echo "Path:         $(TEST_DIR)"	>> $(METADATA)
+	@echo "License:      GPL" >> $(METADATA)
+	@echo "TestVersion:  $(TESTVERSION)"	>> $(METADATA)
+	@echo "Description:  l2tp basic">> $(METADATA)
+	@echo "TestTime:     30m" >> $(METADATA)
+	@echo "RunFor:       $(PACKAGE_NAME)" >> $(METADATA) 
+# add any other packages for which your test ought to run here
+	@echo "Requires:     $(PACKAGE_NAME)" >> $(METADATA)
+	@echo "Requires:     net-tools" >> $(METADATA)
+	@echo "RhtsRequires:    kernel-kernel-networking-common" >> $(METADATA)
```
Comment (rasibley) [inline networking/tunnel/l2tp/basic/Makefile]: RhtsRequires will import internal RH libraries which won't work if running externally, these libraries have already been ported to tests-beaker github repo,  you can instead use RepoRequires like this example: https://github.com/CKI-project/tests-beaker/blob/master/networking/firewall/netfilter/target/Makefile#L84

```
@echo "RepoRequires: networking/common" >> $(METADATA)
@echo "RepoRequires: networking/tunnel/common" >> $(METADATA)
```

**Comment by rasibley** [comment]
@emanchado thanks for the changes, do you have any test jobs you can share ? Feel free to link in the ticket or just add the beaker job ID here.

**Comment by emanchado** [comment]
Oops, that didn't seem to go too well :cry: 

http://lab-02.rhts.eng.rdu.redhat.com/beaker/logs/tasks/98847+/98847012/taskout.log
https://beaker.engineering.redhat.com/recipes/7323116

**Comment by emanchado** [comment]
Cleaned the commits (squashed) and finally successfully ran the tests (on Fedora 30), see job 3775442.

**Pull request merged**